### ISSUE TITLE
No error flagged for hex color missing #

### ISSUE THREAD

**Issue by smlombardi** [stylelint/stylelint#3140]
Shouldn't the `fff` that's missing its `#` be flagged?

```scss
h2 {
  font-size: 24px;
  font-family: $font-family-ps-regular;
  margin-top: 32px;
  margin-bottom: 16px;

  &.ko {
    color: fff;
  }

  &.alt {
    color: $pb-gray-500;
  }
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "at-rule-empty-line-before": null,
    "at-rule-no-unknown": null,
    "declaration-colon-newline-after": null,
    "declaration-empty-line-before": null,
    "max-empty-lines": 2,
    "no-eol-whitespace": [ true, {
        "ignore": ["empty-lines"]
      } ],
    "number-leading-zero": null,
    "rule-empty-line-before": null,
    "selector-list-comma-newline-after": "never-multi-line",
    "selector-list-comma-space-after": "always",
    "selector-pseudo-element-colon-notation": "double",
    "selector-type-no-unknown": null,
    "no-missing-end-of-source-newline": null,
    "block-closing-brace-newline-after": null,
    "unit-whitelist": [
      "em",
      "rem",
      "s",
      "px",
      "%",
      "deg",
      "ms",
      "vh",
      "vw",
      "dppx",
      "dpi",
      "fr"
    ]
  }
}
```

> Which version of stylelint are you using?

    "stylelint": "8.1.1",
    "stylelint-config-standard": "17.0.0"

> How are you running stylelint: CLI, PostCSS plugin, Node API?

In VSCode, in the editor

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

There are no rules covered this case in stylelint. But there is [stylelint-csstree-validator](https://github.com/csstree/stylelint-validator) plugin for checking CSS validity according to specs.

**Comment by smlombardi** [comment]
Thanks, tried it but it flags a lot of errors that aren't technically errors in scss, such as
````
.sequence-map {
  .complete {
    color: theme-color("success");
    fill: theme-color("success");
  }
  .dot {
    fill: $gray-500;
  }
}
````

It doesn't like `color: theme-color("success");`, for example.

**Comment by jeddy3** [comment]
Closing this as [it's best handled](https://github.com/stylelint/stylelint/blob/master/VISION.md#possible-errors) by a parser with syntax validation. 

@smlombardi Please consider helping add syntax extension support to csstree if you'd like to see spec validation that can cope with non-standard syntax like SCSS functions.
